presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/GeometryUtils.java

```diff
@@ -73,6 +76,57 @@ public static GeometryTypeName valueOf(String type)
         }
     }
 
+    // Copy of com.esri.core.geometry.Interop.translateFromAVNaN
+    private static double translateFromAVNaN(double n)
```

Comment:
why is that needed?

Reply:
deserializeEnvelope needs to recognize custom NAN values generated by ESRI's serialization of empty geometries.